test/tests/phi_zero/tests

```diff
@@ -0,0 +1,22 @@
+[Tests]
+  [./phi_zero_linear]
+    type = 'RunApp'
+    input = 'simple_transient_diffusion.i'
+    expect_out = ' Solve Converged!'
```

Comment:
You can remove this. If the solve didn't converge, then this test would register as failed